lib/features/auth/signup/user_information.dart

```diff
@@ -0,0 +1,19 @@
+class UserInformation {
+  String name,
+      emailAddress,
+      instituteName,
+      password,
+      reEnterPassword,
+      countryCode;
+  int phoneNumber;
+
+  void storeData(String s, String keyword) {
+    if (keyword == 'Enter your name') name = s;
```

Comment:
Don't store data like the way you are doing...
there are many faaar better ways to do so
1. constructor
2. setters and getters
3. `UserInformation user = UserInformation();`
`user.name = "required data"`